src/raven.js

```diff
@@ -1950,6 +1951,48 @@ function hasKey(object, key) {
   return objectPrototype.hasOwnProperty.call(object, key);
 }
 
+function matchFilters(filters, object) {
```

Comment:
Can you please match code style with the rest of the codebase?
Mostly multiple vars and index inside `for` statement.